website/docs/guides/migrating-between-renamed-resources.html.markdown

```diff
@@ -9,20 +9,17 @@ description: |-
 
```

Comment:
I went a head and made changes here to support future versions. Let me know your thoughts @katbyte